crowdin.yaml

```diff
@@ -1,19 +1,3 @@
-project_identifier: django-girls-tutorial
-api_key: 2be4b1707d0745bc96e9a5c9ffb4ca9b
-base_path: /Users/olasitarska/Desktop/Events/Django Girls/tutorial
```

Comment:
I believe that is correct path on @olasitarska environment and was used with crowdin-cli (actually I believe, that whole content of this file was generated using `crowding generate` command) and doesn't make sense beside her environment.

See: https://support.crowdin.com/cli-tool/#configuration